packages/readabilityjs/Readability.js

```diff
@@ -2833,6 +2835,11 @@ Readability.prototype = {
     return false;
   },
 
+  _getLanguage: function(code) {
+    let lang = new Intl.DisplayNames(['en'], {type: 'language'});
```

Comment:
nice, is `en` the fallback value here?

Reply:
`en` here is the locale of the display name so the display names would be in English

Reply:
I have added a fallback value = 'English'